validator/slashing-protection/local/standard-protection-format/import_test.go

```diff
@@ -0,0 +1,213 @@
+package interchangeformat
+
+import (
+	"bytes"
+	"context"
+	"encoding/hex"
+	"encoding/json"
+	"fmt"
+	"strconv"
+	"testing"
+
+	"github.com/prysmaticlabs/prysm/shared/bls"
+	"github.com/prysmaticlabs/prysm/shared/bytesutil"
+	"github.com/prysmaticlabs/prysm/shared/params"
+	"github.com/prysmaticlabs/prysm/shared/rand"
+	"github.com/prysmaticlabs/prysm/shared/testutil/assert"
+	"github.com/prysmaticlabs/prysm/shared/testutil/require"
+	"github.com/prysmaticlabs/prysm/validator/db/kv"
+	dbtest "github.com/prysmaticlabs/prysm/validator/db/testing"
+)
+
+const numValidators = 10
+
+func TestStore_ImportInterchangeData_BadJSON(t *testing.T) {
+	ctx := context.Background()
+	publicKeys := createRandomPubKeys(t)
+	validatorDB := dbtest.SetupDB(t, publicKeys)
+
+	buf := bytes.NewBuffer([]byte("helloworld"))
+	err := ImportStandardProtectionJSON(ctx, validatorDB, buf)
+	require.ErrorContains(t, "could not unmarshal slashing protection JSON file", err)
+}
+
+func TestStore_ImportInterchangeData_BadFormat_PreventsDBWrites(t *testing.T) {
```

Comment:
Also might be nice to add in one more test case for what is the max value for which we can store unsigned integers as a string. 
https://golang.org/src/strconv/atoi.go?s=1855:1918#L109

Reply:
Added a test to prove we can parse MaxUint64. Anything beyond that throws an error